main.go

```diff
@@ -6,12 +6,14 @@ import (
 	"github.com/os-observability/tempo-operator/cmd"
 	"github.com/os-observability/tempo-operator/cmd/generate"
 	"github.com/os-observability/tempo-operator/cmd/start"
+	"github.com/os-observability/tempo-operator/cmd/version"
 )
 
 func main() {
 	rootCmd := cmd.NewRootCommand()
 	rootCmd.AddCommand(start.NewStartCommand())
 	rootCmd.AddCommand(generate.NewGenerateCommand())
+	rootCmd.AddCommand(version.NewVersionCommand())
```

Comment:
How is the tempo version propagated to the version command? 

The tempo version can change and might change at runtime. 

We should probably as well print the operator config at the operator start.

Reply:
> How is the tempo version propagated to the version command?
> 
> The tempo version can change and might change at runtime.
I prepared it to be configured at build time. As in the Jaeger Operator. From the @andreasgerstmayr comments, I'll change how the version is get.

> We should probably as well print the operator config at the operator start.

I did it [cmd/start/main.go](https://github.com/os-observability/tempo-operator/pull/184/files#diff-12831ff141f9eed16d3dc6aefbf0827971668f73f16bd3ba00ca26c69bbb9299).

Reply:
> The tempo version can change and might change at runtime.

What is our support policy on that? I'd assume we only support the version of the default image as shipped with the default config, and if users use a custom image they need to make sure it's compatible (e.g. tempo v2.0 has breaking changes in the config file, I don't think we should support multiple tempo versions in the same operator version).

Reply:
+1 @andreasgerstmayr